include/mbgl/ios/MGLAnnotationImage.h

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
```

Comment:
No, but then this header should pull in MGLTypes.h. Headers shouldn't have implicit dependencies.